api_app/api/routes/airlock.py

```diff
@@ -15,15 +15,15 @@
 from db.repositories.airlock_requests import AirlockRequestRepository
 from models.schemas.airlock_request import AirlockRequestInCreate, AirlockRequestInResponse
 from resources import strings
-from services.authentication import get_current_workspace_owner_or_researcher_user, get_current_workspace_owner_user
+from services.authentication import get_current_workspace_owner_or_researcher_user_or_airlock_manager, get_current_workspace_owner_or_researcher_user, get_current_airlock_manager_user
```

Comment:
I don't see you use the get_current_workspace_owner_or_researcher_user_or_airlock_manager anywhere, also not sure it is needed

Reply:
On line 26, it is necessary